service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -95,20 +96,21 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
         }
 
         if (isNotEmpty(caseData.getAllocatedJudge())) {
-            caseDetails.getData().put(JUDGE_AND_LEGAL_ADVISOR_KEY, setAllocatedJudgeLabel(caseData));
+            caseDetails.getData().put(JUDGE_AND_LEGAL_ADVISOR_KEY, prepareJudgePage(caseData));
         }
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(caseDetails.getData())
             .build();
     }
 
-    private JudgeAndLegalAdvisor setAllocatedJudgeLabel(CaseData caseData) {
+    private JudgeAndLegalAdvisor prepareJudgePage(CaseData caseData) {
         JudgeAndLegalAdvisor judgeAndLegalAdvisor = JudgeAndLegalAdvisor.builder().build();
 
         if (isNotEmpty(caseData.getStandardDirectionOrder())
             && isNotEmpty(caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor())) {
```

Comment:
Do you think it's worthwhile creating a boolean private method from this? Or perhaps on the SDO model itself we could have `boolean hasJudge()`.

Reply:
Probably not worthwhile as this is the only time we are checking for a judge, but good to keep in mind